pkg/apis/ceph.rook.io/v1/template/scc.yaml

```diff
@@ -0,0 +1,40 @@
+kind: SecurityContextConstraints
+apiVersion: v1
+metadata:
+  name: rook-ceph
+allowPrivilegedContainer: true
+# un-comment if host networking will be enabled
+# allowHostNetwork: true
+# allowHostPorts: true
+priority:
+allowedCapabilities: []
+# un-comment if the cluster will be encrypted
+# allowHostIPC: true
+readOnlyRootFilesystem: false
+requiredDropCapabilities: []
+defaultAddCapabilities: []
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: MustRunAs
+fsGroup:
+  type: MustRunAs
+supplementalGroups:
+  type: RunAsAny
+volumes:
+  - configMap
+  - downwardAPI
+  - emptyDir
+  - flexVolume
+  - hostPath
+  - persistentVolumeClaim
+  - projected
+  - secret
+users:
+  # A user needs to be added for each rook service account.
+  # This assumes running in the default sample "rook-ceph" namespace.
+  # If other namespaces or service accounts are configured, they need to be updated here.
+  - system:serviceaccount:rook-ceph:rook-ceph-system # serviceaccount:namespace:operator
```

Comment:
same here as well namespace should be configurable?

Reply:
Good point.